src/System.Diagnostics.Process/tests/ProcessTests.netstandard.cs

```diff
@@ -66,7 +66,7 @@ public void TestRespondingWindows()
             }
         }
 
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Uses P/Invokes to get the process Id
```

Comment:
This test does not contain any direct P/Invokes.  It only invokes members of Process.  Are you saying that, on Windows, this test calls to that Unix P/Invoke you mention?  If so, that's a huge bug in the implementation.  If not, then the comment isn't accurate.